renderer.js

```diff
@@ -1,8 +1,18 @@
-Mousetrap.bind('shift+q', function() {
+Mousetrap.bind('ctrl+shift+q', function() {
     window.closeCurrentWindow();
 });
+
+colorMap = {0: "red", 1: "orange", 2: "gold", 3: "green", 4: "blue", 5: "indigo", 6: "violet"}
+
 document.body.addEventListener('keydown', function(e) {
     e.preventDefault();
-    document.getElementById("key").innerHTML = e.key;
+    if (e.key.length < 2){
+        charCode = e.key.charCodeAt(0);
+        document.getElementById("key").innerHTML = e.key;
```

Comment:
Let's use either double quotes or single quotes. It looks like I started with single quotes so we can stick with that.

Reply:
Switched - though it looks like your "key" was in double quotes.